src/modules/Auth/features/Login/LoginForm/LoginForm.tsx

```diff
@@ -33,6 +33,10 @@ export const LoginForm = () => {
     defaultValues: { email: '', password: '' },
   });
 
+  useEffect(() => {
+    Mixpanel.trackPageView('Login');
+  }, []);
```

Comment:
All effects should be declared at the end of a function body, just before the `return` statement

Reply:
Actually, it would be great if you moved to effect to the page component